src/CMakeLists.txt

```diff
@@ -80,9 +77,20 @@ endif()
 get_target_property(ESPP_CFLAGS _espressopp COMPILE_OPTIONS)
 message(STATUS "_espressopp compiler flags are: ${ESPP_CFLAGS}")
 
-target_link_libraries(_espressopp ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} MPI::MPI_CXX ${FFTW3_LIBRARIES} ${VAMPIRTRACE_LIBRARIES})
+target_include_directories(_espressopp PUBLIC ${CMAKE_SOURCE_DIR}/src)
```

Comment:
All these 3 should be marked e.g. `$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/src>` as they aren't useful after install.